apps/frontend/src/components/ui/ProjectMemberHeader.vue

```diff
@@ -336,6 +336,7 @@ const nags = computed(() => [
     },
   },
   {
+    hide: props.project.stats === "draft",
     condition: props.tags.rejectedStatuses.includes(props.project.status),
```

Comment:
wait, this should already be hidden by the condition, no?  Draft should not be a rejected status